src/prefect/agent/agent.py

```diff
@@ -99,10 +98,7 @@ def __init__(
     ) -> None:
         self.name = name or config.cloud.agent.get("name", "agent")
 
-        self.labels = labels or config.cloud.agent.get("labels", [])
-        # quick hack in case config has not been evaluated to a list yet
-        if isinstance(self.labels, str):
-            self.labels = ast.literal_eval(self.labels)
+        self.labels = labels or list(config.cloud.agent.get("labels", []))
```

Comment:
I found some failing tests that were being caused by this manipulating the _same_ list with each test run, hence the 'list()` wrapper.